satpy/readers/ahi_hsd.py

```diff
@@ -255,12 +255,38 @@ class AHIHSDFileHandler(BaseFileHandler):
                             reader_kwargs={'calib_mode':: 'update'})
         scene.load([0.6])
 
+    Alternative AHI calibrations are also available, such as GSICS
+    coefficients. As such, you can supply custom per-channel calibration
+    for the infrared channels by setting calib_mode='custom' and passing
```

Comment:
> Just want to check I haven't left a reference to IR-only somewhere!

@simonrp84 Here is a reference to IR channels, but maybe I just misread it, because GSICS is IR only

Reply:
Should be fixed now!